### DISCUSSION TITLE
mspider/install/install/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1189] [Announcements]
# mspider/install/install/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/install/install/

**Comment by liyuerich** [comment]
安装的命令都是在线环境，如果客户是离线环境的话，安装不成功。 能否提供离线环境命令？

**Reply by samzong** [reply]
网格暂时不提供独立安装的内容，后面这篇文档我们调整下。 @huan555

**Comment by kebe7jun** [comment]
需要增加一些常见的参数说明。
